samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb

```diff
@@ -9,13 +9,13 @@
     "In this notebook you will create e2e mnist Kubeflow Pipeline to perfom:\n",
     "- Hyperparameter tuning using Katib\n",
     "- Distributive training with the best hyperparameters using TFJob\n",
-    "- Serve the trained model using KFServing\n",
+    "- Serve the trained model using KServe\n",
     "\n",
     "Reference documentation:\n",
     "\n",
     "- https://www.kubeflow.org/docs/components/training/tftraining/\n",
     "- https://www.kubeflow.org/docs/components/katib/\n",
-    "- https://www.kubeflow.org/docs/components/kfserving/\n",
+    "- https://www.kubeflow.org/docs/external-add-ons/kserve/\n",
     "\n",
     "**Note**: This Pipeline runs in the multi-user mode. Follow [this guide](https://github.com/kubeflow/katib/tree/master/examples/v1beta1/kubeflow-pipelines#multi-user-pipelines-setup) to give your Notebook access to Kubeflow Pipelines."
```

Comment:
I think the guide should be https://www.kubeflow.org/docs/components/pipelines/sdk/connect-api/#multi-user-mode.